src/data/set/lattice.lean

```diff
@@ -3,14 +3,47 @@ Copyright (c) 2014 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad, Leonardo de Moura, Johannes Hölzl, Mario Carneiro
 -/
-import order.complete_boolean_algebra
 import data.sigma.basic
-import order.galois_connection
+import order.complete_boolean_algebra
 import order.directed
+import order.galois_connection
+
+/-!
+# The set lattice
+
+This file provides usual set notation for unions and intersections, a `complete_lattice` instance
+for `set α`, and some more set constructions.
+
+## Main declarations
+
+* `set.lattice_set`: `set α` is a `complete_lattice` with `< = ⊂`, `≤ = ⊆`, `⊓ = ∩`, `⊔ = ∪`. See
```

Comment:
I think mentioning `` `⨅ = ⋂` and`⨆ = ⋃` `` is important here, because these are the only two operations that this file actually provides, and neither of them are the ones you mention!